tests/WebpackNetteAdapter/DevServerTest.phpt

```diff
@@ -38,7 +38,7 @@ class DevServerTest extends TestCase
 
 	public function testDevServer(): void
 	{
-		$devServer = new DevServer(TRUE, 'http://localhost:3000', 0.1, $this->httpClient);
+		$devServer = new DevServer(TRUE, 'http://localhost:3000', '', 0.1, $this->httpClient);
```

Comment:
Could you please add a test with a proxy URL set, and test that

1. the heartbeat request still hits the original URL, and
2. `getUrl()` returns the proxied one?

I know it's trivial but I'd prefer to be safe rather than sorry :)

Reply:
Sure, check it here https://github.com/o2ps/WebpackNetteAdapter/pull/13/commits/213eb0de46886b43131156dabafb511166f1c52b